Re-ran targeted verification on branch `bugfix/session-review-helper-quality-debt-4086` after the CHANGES_REQUESTED feedback was addressed in commits a1571e4, 79187978, and e9952dd3.

Commands run:
- `shellcheck -e SC1091 .agents/scripts/session-review-helper.sh`
- `bash .agents/scripts/session-review-helper.sh security --json >/dev/null`
- `bash .agents/scripts/session-review-helper.sh security --json --session pid-21760 >/dev/null`
- `bash .agents/scripts/session-review-helper.sh security --session 'bad$session' >/dev/null 2>&1` (expected non-zero)

Results:
- ShellCheck: pass
- security --json: pass (valid JSON emitted)
- security --json --session pid-21760: pass (valid JSON emitted; `global_only: true` present on global sections)
- invalid session negative test: pass (command exited non-zero as expected)

--- a/.agents/scripts/session-review-helper.sh
+++ b/.agents/scripts/session-review-helper.sh
@@ -681,9 +681,7 @@ _security_summary_json() {
 		else
 			context_result=$("$session_helper" get-context 2>/dev/null || echo "")
 		fi
-		if [[ -n "$context_result" ]]; then
-			session_context_json=$(printf '%s' "$context_result" | jq -c --argjson available true '. + {available: $available}' 2>/dev/null || echo '{"available":false}')
-		fi
+		session_context_json=$(printf '%s' "$context_result" | jq -c --argjson available true '. + {available: $available}' 2>/dev/null || echo '{"available":false}')
 	fi
 
 	# Quarantine — query helper if available

